debian/copyright

```diff
@@ -1,5 +1,50 @@
-© 1993‒2021 Marko Mäkelä and others.
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cbmconvert
+Upstream-Contact: https://github.com/dr-m/cbmconvert/issues
+Source: https://github.com/dr-m/cbmconvert
+License: GPL-2+
 
-According to the file "COPYING" all parts of this package are licenced
-under the terms of the GNU GPL Version 2 of which a copy is available
-in /usr/share/common-licenses.
+Files: *
+Copyright: (C) 1993‒2021 Marko Mäkelä <marko.makela at iki.fi>
+License: GPL-2+
+
+Files: image.c
+Copyright: (C) Pasi Ojala <albert at cs.tut.fi>,
+ (C) 1993‒2021 Marko Mäkelä <marko.makela at iki.fi>
+License: GPL-2+
+
+Files: t64.c
+Copyright: (C) 1995 Daniel Fandrich,
+ (C) 1997-2021 Marko Mäkelä <marko.makela at iki.fi>
```

Comment:
The previous lines use U+2012 FIGURE DASH for the year ranges, while this and the following lines use U+002D.

Reply:
Dashes should be OK now. I think I just copied those from source and not recognized the difference.
Your CPACK config also updated.